lane.py

```diff
@@ -2454,6 +2483,22 @@ def uses_customlists(self):
             return True
         return False
 
+    def max_cache_age(self, type):
+        """Determine how long a feed for this WorkList should be cached
+        internally.
+
+        :param type: The type of feed.
+        """
+        if type == CachedFeed.GROUPS_TYPE:
+            # Generating grouped feeds on the fly for Lanes is not incredibly
+            # expensive, but it's slow enough that we prefer to regenerate
+            # them in the background (using force_refresh=True) rather
+            # than while someone is waiting for an HTTP response.
+            return CachedFeed.CACHE_FOREVER
```

Comment:
If grouped feeds will be generated in the background, then why return "CACHE_FOREVER"? Or is that used along with force_refresh=True?

Reply:
CACHE_FOREVER is how `_should_refresh` knows not to say "yes, we should refresh", even if the thing in the cache is really really old.